test/system/users_test.rb

```diff
@@ -34,4 +34,15 @@ class UsersTest < ApplicationSystemTestCase
     click_on 'Sign out'
     assert_selector 'h2', text: 'Log in'
   end
+
+  test 'user can sign in with google account' do
+    visit new_user_session_path
+
+    click_on 'Sign in with Google'
+
+    assert_equal user_google_oauth2_omniauth_authorize_path, current_path
+    get '/users/auth/google_oauth2' # here user should actually sign in
```

Comment:
The problem is, that we can't actually simulate sign in with google: we can't access any selector on Google consent screen and click on it in test. So we should simulate login with Google maybe using mock. 
I have seen two examples in Apps app, but both for some reason don't work for me:
in `test/test_helper.rb`
```
# einstein is mocked in config/initializers/omniauth.rb
  def sign_in_as_einstein
    get "/auth/google_oauth2"
    follow_redirect!
  end
```

and in `test/application_system_test_case.rb`:

```
 def sign_in
    visit '/auth/google_oauth2'
  end
```

But m.b. you would have an idea how to approach it?